nativescript-core/Application.md

```diff
@@ -0,0 +1,690 @@
+---
+title: Application
+---
+
+# Application
+
+The Application module provides abstraction over the platform-specific application implementations. The module lets you manage things such as handling lifecycle events of your application (cross-platform and/or native), sending Broadcasts on Android or adding a Notification observer on IOS.
+
+### Import
+
+/// flavor javascript
+
+```javascript
+import { Application } from '@nativescript/core'
+```
+
+///
+
+/// flavor typescript
+
+```typescript
+import { Application } from '@nativescript/core'
+```
+
+///
```

Comment:
```suggestion
```typescript
import { Application } from '@nativescript/core'
```
```

We don't really need to have a flavor switcher in this case, as both of these are identical.